readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: Viper007Bond, automattic, donncha
 Donate link: https://alex.blog/2019/03/13/in-memory-of-alex-donation-link-update/
 Tags: code, source, sourcecode, php, syntax highlighting, syntax, highlight, highlighting, highlighter, WordPress.com
 Requires at least: 5.7
```

Comment:
Shouldn't we bump this too? :eyes:

Reply:
I think for this case that we're not introducing new features, we don't need to bump it. And we make sure that it's a change with less noise for the users using it too. :)